WebHostLib/templates/options.yaml

```diff
@@ -49,6 +49,8 @@ requires:
     {% if option.default == "random" %}
     random: 50
     {%- endif -%}
+    {%- elif is_string(option.default) %}
+    {{ option.default }}: 50
     {%- else %}
     {{ yaml_dump(default_converter(option.default)) | indent(4, first=False) }}
```

Comment:
Mentioning it here again, so we don't forget: I think moving the default-is-string-handling to `default_converter` would be better.

`{{ yaml_dump({option.default: 50}) }}` produces the same as `{{ option.default }}: 50`, so that should be easy.